airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/api.py

```diff
@@ -87,10 +88,19 @@ def _parse_call_rate_header(headers):
 
         return usage, pause_interval
 
+    def compute_pause_interval(self, usage, pause_interval):
+        """ The sleep time will be calculated based on usage consumed.
+        """
+        max_pause_interval = self.MIN_PAUSE_INTERVAL
+        if usage >= self.MAX_RATE:
+            max_pause_interval = self.MAX_PAUSE_INTERVAL
+
+        return max(max_pause_interval, pause_interval)
+
     def handle_call_rate_limit(self, response, params):
         if "batch" in params:
             max_usage = 0
-            max_pause_interval = self.pause_interval_minimum
+            max_pause_interval = self.MIN_PAUSE_INTERVAL
```

Comment:
Could you add a comment or find a new variable naming that could help not thinking this line has a typo: setting a max value to a min value is awkward.